docs/getting-started/invite-members/README.md

```diff
@@ -64,13 +65,13 @@ Type in the email address of the new member. If OpenProject does not find an exi
 
 An email invitation will be sent out to the user with a link to [create an account](../sign-in-registration/#create-a-new-account) for OpenProject.
 
-![invite-new-members](1566224961670.png)
+![invite-members-via-email](Bildschirmfoto 2022-03-02 um 14.19.56.png)
 
 **Invite from header**
 
-In the header menu that is always present, you find a green button that allows you to invite users from wherever in the application you are.
+In the header menu that is always present, you find a green button that allows you to create projectes, work packages an invite users from wherever in the application you are.
 
-![invite-user-home](invite-user-home.png)
+![create-project-button](Bildschirmfoto 2022-03-02 um 14.26.22.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.